domain/src/test/java/org/oppia/domain/classify/rules/dragAndDropSortInput/DragDropSortInputIsEqualToOrderingWithOneItemIncorrectClassifierProviderTest.kt

```diff
@@ -98,9 +98,7 @@ class DragDropSortInputIsEqualToOrderingWithOneItemIncorrectClassifierProviderTe
   }
 
   @Test
-    /* ktlint-disable max-line-length */
-  fun testAnswer_testLisOfSetsOfHtmlString_differByOneElement_differentOrder_bothValuesDoNotMatch_failsCorrectly() {
-    /* ktlint-enable max-line-length */
+  fun testAnswer_testLisOfSetsOfHtmlString_differByOneElement_differentOrder_bothValuesDoNotMatch_failsCorrectly() {  /* ktlint-disable max-line-length */
```

Comment:
Ktlint does not take it as a one-line comment, it's like disabling starts from the line below it till where you mention about enable. Use `// ktlint-disable max-line-length` as mentioned earlier. Also not put extra space before the comment, only one space else the ktlint will fail. 
I suggest please go through the [wiki page](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide) once.

Reply:
okay got it thanks for the clarification 👍

Reply:
Seems it is done.